includes/cf-descriptions/atmos-undefined.md

```diff
@@ -0,0 +1,6 @@
+**ATMOS (undefined)**<br>
+
+Handles cases where only Atmos is specified in title but not DD+ or TrueHD (Where it is not specified if it is Lossy or Lossless)
+
+!!! note
+    Give this the same score as Lossy Atmos, and then on import, it will get changed to either lossy or lossless based on mediainfo.
```

Comment:
```suggestion
    Apply the same score as Lossy (DD+) Atmos. On import, it will be updated to reflect either lossy or lossless based on the media's info using ffprobe.
```